packages/verifiable-credentials/test/share-credentials.test.ts

```diff
@@ -37,11 +37,14 @@ describe('Share Credential tests', function () {
                 data.result.id
             );
 
-            const uriWithoutKey = data.uri.substring(0, expectedUri.length);
+            const base64decodedURI = base64Encoded(data.veridaUri)
```

Comment:
`base64decodedURI` should be `base64encodedURI`

Reply:
since , I'm not i won't base64encode the `veridaUrl` i don't think i will need to use this `base64Encoded` anymore